frame/wormhole/backing/s2s/src/tests.rs

```diff
@@ -158,11 +178,12 @@ impl Config for Test {
 
 	type OutboundPayload = ();
 	type CallEncoder = MockCallEncoder;
+	type MessageNoncer = MockLatestMessageNoncer;
 
-	type FeeAccount = ();
-	type MessageSender = MockRelayCaller;
 	type MessageSendPalletIndex = BridgePangolinIndex;
 	type MessageLaneId = BridgePangolinLaneId;
+	type OutboundMessageFee = Balance;
```

Comment:
Another question: `MessageSendPalletIndex` this config is useless? Maybe we can remove it.

Reply:
Yes exactly